pkg/envoy/ingress_translator.go

```diff
@@ -154,7 +154,7 @@ func classFilter(ingresses []v1.Ingress, ingressClass []string) []v1.Ingress {
 				if i.GetAnnotations()["kubernetes.io/ingress.class"] == class || *i.Spec.IngressClassName == class {
 					is = append(is, i)
 				} else {
-					logrus.Debugf("the ingress class of %s is not %d", i.Name, &class)
+					logrus.Debugf("the ingress class of %s is not %d", i.Name, class)
```

Comment:
```suggestion
					logrus.Debugf("the ingress class of %s is not %s", i.Name, class)
```
%d is for decimal integers and `class` is a string